common/src/Utilities/Windows/WinMisc.cpp

```diff
@@ -108,7 +103,7 @@ wxString GetOSVersionString()
     else
         GetSystemInfo(&si);
 
-    if (VER_PLATFORM_WIN32_NT != osvi.dwPlatformId || osvi.dwMajorVersion <= 5)
+    if (VER_PLATFORM_WIN32_NT != osvi.dwPlatformId || osvi.dwMajorVersion <= 5 || (osvi.dwMajorVersion == 6 && osvi.dwMinorVersion <= 2))
```

Comment:
Just use `VerifyVersionInfo` and save yourself the trouble of comparing version numbers manually.